### PR TITLE
More docs fixes

### PR DESCRIPTION
**Pull request by commit111**
- More consistent wording between related pages (e.g. Managed Redis and Managed Postgres)
- More cross-references + fix broken links
- Removed outdated wordings + typos

### CODE REVIEW

File: docs/concepts/accounts.md [hunk 1]
```diff
@@ -6,32 +6,28 @@ sidebar_position: 100
...
 # Accounts
 
-In this section, we'll be talking about a couple different topics related to accounts in Defang. First we'll cover authentication and how to create an account, and then we'll talk about how resources are organized within a cloud environment using your account info.
+Below, we will cover account creation with Defang, and how resources are organized within a cloud environment using your account info.
 
-:::tip Why do I need an account?
-Defang requires an account so we can organize your resources and provide you with a way to manage them. We also use your account to authenticate you when you interact with [Defang Playground](./defang-playground.md) and the [Defang Portal](./portal.md).
+Defang requires an account so we can organize your resources and provide you with a way to manage them. We also use your account to authenticate you when you interact with [Defang Playground](./defang-playground.md) and the [Defang Portal](./portal.md). 
 
 Eventually, billing and payment information will be associated with your account.
```
Comment (raphaeltm) [inline docs/concepts/accounts.md:13]: @commit111 I think we can also remove this line, because billing is now associated with an account.

**Review by raphaeltm — CHANGES_REQUESTED** [review]

File: docs/concepts/accounts.md (hunk 1 above)
Comment (raphaeltm) [reply docs/concepts/accounts.md]: @commit111 I've updated that line. Let me know what you think and we can merge this.

File: docs/concepts/deployments.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ When you deploy using Defang, whether it's with `defang compose up` with a [Comp
 
 ## Deploying Updates
 
-When you run a deployment to update one or more services, Defang will also make sure to get your new services up and running before deprovisioning any old services so you don't have to worry about downtime.
+When you run a deployment to update one or more [services](/docs/concepts/services), Defang will also make sure to get your new services up and running before deprovisioning any old services so you don't have to worry about downtime.
```
Comment (jordanstephens) [inline docs/concepts/deployments.md:13]: This is only true for the `production` deployment mode. The following suggestion briefly describes an update deployment, then calls out zero-downtime deployments with the `production` mode.

```suggestion
When you run a deployment to update one or more [services](/docs/concepts/services), Defang will build new images for your services, and provision new resources to replace your existing services.

### Zero Downtime Deployments

Defang can deploy your services using different [modes](/docs/concepts/deployment-modes). When using the `production` mode, Defang will make sure the new replacement services are healthy before deprovisioning your existing services.

By default, using the `development` mode, defang will deprovision your existing services before provisioning replacements. This helps reduce costs.
```

File: docs/concepts/deployments.md [hunk 3]
```diff
@@ -10,12 +10,14 @@ When you deploy using Defang, whether it's with `defang compose up` with a [Comp
...
 
 :::info
-In [Defang BYOC](./defang-byoc.md), Defang will use your cloud provider account to build and store your images. In [Defang Playground](./defang-playground.md) we will build and store your images for you.
+In [Defang BYOC](./defang-byoc.md), Defang uses your cloud provider account to build and store your images. In [Defang Playground](./defang-playground.md), we build and store your images for you.
 :::
 
+Note that we offer different [deployment modes](/docs/concepts/deployment-modes): development, staging, and production. You can switch the modes using the CLI. 
+
```
Comment (jordanstephens) [inline docs/concepts/deployments.md:20]: I would put a header above this, and maybe preface it with "as mentioned above" to make it sound less redundant.

```suggestion

### Deployment Modes

As mentioned above, Defang offers different [deployment modes](/docs/concepts/deployment-modes): `development`, `staging`, and `production`. You can switch the modes using the `--mode` cli flag.
```

**Review by jordanstephens — APPROVED** [review]
Two suggestions inline, but this is a great step forward. Thanks Linda!

File: docs/concepts/accounts.md (hunk 1 above)
Comment (commit111) [reply docs/concepts/accounts.md]: thanks! LGTM

**Pull request merged**